operator-framework/operator-lifecycle-manager
# Add Uninstall feature, add Makefile commands and organize docs

## Motivation
https://github.com/operator-framework/operator-lifecycle-manager/issues/896
https://github.com/operator-framework/operator-lifecycle-manager/issues/789

## What
- Add command `make uninstall`
- Add make commands for manual/local installation
- Organize information into the README. MD and doc/install.md
- Add github files (PR/Issue templates) 
- Add CONTRIBUTION.MD doc
- Replace the directory "Documentations" for `doc` in order to follow the project and org standards

## Why
- Adding missing steps to install
- Make the info more easy to understand and to find
- Follow the org standards 
- Asked for the required info when someone opens an issue
- Give a better user experience 

## Steps to verify
- Check the [README](https://github.com/operator-framework/operator-lifecycle-manager/blob/049732efc04b921dee58324a80a1d505863584a3/README.md)
- Check the [install.md doc](https://github.com/operator-framework/operator-lifecycle-manager/blob/7cce6020fb0a24c091f4ec9f3af789d0115310e3/doc/install/install.md)
- Run `make uninstall` with a local installation
